engine/src/main/java/org/terasology/world/chunks/internal/ChunkImpl.java

```diff
@@ -64,6 +65,9 @@
     private static final DecimalFormat PERCENT_FORMAT = new DecimalFormat("0.##");
     private static final DecimalFormat SIZE_FORMAT = new DecimalFormat("#,###");
 
+    private final int verticalChunkMeshSegments = CoreRegistry.get(Config.class).getSystem().getVerticalChunkMeshSegments();
+    private final int chunkHalfHeight = ChunkConstants.SIZE_Y / verticalChunkMeshSegments / 2;
```

Comment:
I propose we just remove the whole vertical chunk thing, I don't think it is needed.

Effectively it just causes chunks to be subdivided and meshes to be created for each subdivision. But the default is 1 and I doubt anyone is changing it.